src/app/shared/navbar/searchbar/searchbar.html

```diff
@@ -0,0 +1,16 @@
+<div class="search-input-container">
+  <input
+    placeholder="Search"
+    type="text"
+    (focus)="toggleIsExpanded(true)"
+    (blur)="toggleIsExpanded(false)"
+    (keyup.enter)="navigate($event.target.value.toLowerCase())"
+    [mdAutocomplete]="auto"
+    [formControl]="searchControl">
+</div>
+
+<md-autocomplete #auto="mdAutocomplete">
+  <md-option *ngFor="let item of filteredSuggestions | async" [value]="item.name">
```

Comment:
Another idea would be to bind the entire item as the option value, and use the `displayWith` property on `md-autocomplete` to map that object to its display value (the name). That way, you don't have to go through the list in `navigate()` to find the ID that matches the search string. You can just get the object itself and pass through its ID.